src/math/ndarray.ts

```diff
@@ -161,6 +161,12 @@ export class NDArray<R extends Rank = Rank> {
     return ops.randUniform(shape, a, b, dtype);
   }
 
+  static range(
```

Comment:
It might be that this isn't in the right place as a number of other array creation methods right above this are indicated as deprecated. Lets chat about where this should be. I guess this would also affect how the tests are called.